docs/process/labels.md

```diff
@@ -0,0 +1,121 @@
+| [:mag_right:][more-information-needed] | `more-information-needed` | The submitter needs to provide more information about the issue |
+| [:mag_right:][priority-1]              | `priority-1`              | Major bug affecting large population and inhibiting their work |
+| [:mag_right:][priority-2]              | `priority-2`              | Bug that affects more than a few users in a meaningful way but doesn't prevent core functions |
+| [:mag_right:][priority-3]              | `priority-3`              | Bugs that affect small number of users and/or relatively cosmetic in nature |
+
+### External contributions
+
+We use these labels to identify work that is ideal for external contributors to
+get involved with.
+
+|                                 | Label name         |  Description |
+| ------------------------------- | ------------------ |  ----------- |
+| [:mag_right:][good first issue] | `good first issue` | Issues marked as ideal for a brand new contributor to start with |
+| [:mag_right:][help wanted]      | `help wanted`      | Issues marked as ideal for external contributors |
+
+### Planning
+
+We use these labels to track tasks outside the usual flow of addressing bugs or
+implementing features:
+
+|                                   | Label name           |  Description |
+| --------------------------------- | -------------------- |  ----------- |
+| [:mag_right:][meta]               | `meta`               | Issues used to co-ordinate tasks or discuss a feature before the required work is captured |
+| [:mag_right:][user-research]      | `user-research`      | Issues that may benefit from user interviews, validations, and/or usability testing |
+| [:mag_right:][needs-design-input] | `needs-design-input` | Issues that require design input from the core team before the work can be started |
+
+### Specialized areas
+
+We use these labels to identify issues related to a specific area or the app,
+or a specific subset of users:
+
+|                             | Label name     | Description |
+| --------------------------- | -------------- | ----------- |
+| [:mag_right:][codemirror]   | `codemirror`   | Issues related to our use of [CodeMirror](https://codemirror.net/) that may require upstream fixes |
+| [:mag_right:][electron]     | `electron`     | Issues related to our use of [Electron](https://electronjs.org) that may need updates to Electron or upstream fixes |
+| [:mag_right:][integrations] | `integrations` | Issues related to editor and shell integrations that ship in Desktop |
+| [:mag_right:][performance]  | `performance`  | Relating to things affecting performance |
+| [:mag_right:][themes]       | `themes`       | Issues related the light or dark themes that ship in Desktop |
+| [:mag_right:][website]      | `website`      | Issues that relate to external websites and require co-ordination to resolve |
```

Comment:
Only two items here don't follow the `Issues related to ` format. Consistency? `Issues` is probably redundant here given the description above

Reply:
See my previous point ⬆️